calypso/proto.go

```diff
@@ -60,14 +60,16 @@ type Read struct {
 // These are the messages used in the API-calls
 // ***
 
-// Authorise is used to add the given ByzCoinID into the list of
-// authorised IDs.
-type Authorise struct {
```

Comment:
The change is right but it breaks backwards compatibility. Should this one be deprecated but still exist ?

Reply:
I let this up to DEDIS to decide - I'm not aware of any active project that uses this API-endpoint, so I took the liberty to change the name. But if you want to keep the exact naming, I will change it back to `Authorise`.

Reply:
@jeffallen ?
You could also mark them as deprecated, and create a deprecated handler that is listening on `Authorise` that can be cleaned up in v4.

Reply:
Sorry to be absent for so long, please let's respect our backwards compatibility commitment strongly for Go server-side things. I'm ok with a bit more flux on client-side APIs that are very immature.

Reply:
Thumbs-up -> did it...